alarm.config

```diff
@@ -27,15 +44,48 @@ wind=1
 # wind is available only with metric
 wind_chill=1
 # default set from November - March
+host=query.yahooapis.com
+path=/v1/public/yql?q=select%%20*%%20from%%20weather.forecast%%20where%%20woeid%%3D
+pathtail=&format=json
 
 [btc]
 enabled=1
+stype=content
+host=coinbase.com
+path=/api/v1/prices/buy
 
 [stocks]
 enabled=1
+stype=content
 tickers=fb,pot.nz
+host=query.yahooapis.com
+path=/v1/public/yql?q=select%%20*%%20from%%20yahoo.finance.quote%%20where%%20symbol%%20in%%20(%%27
+pathtail=%%27)%%20&format=json&env=store://datatables.org/alltableswithkeys
 
 [news]
 enabled=1
+stype=content
+host=feeds.bbci.co.uk
+path=/news/world/rss.xml
 
+## TTS engines
+# NB: Order implies preference for enabled tts engines
+[trygoogle]
+enabled=1
+stype=tts
+head=wget -q -U Mozilla 
+host=translate.google.com
+path=/translate_tts
+lang=en
+client=tw-ob
+tail=.mp3
+player=mpg123 -g 100 -h 10 -d 11
 
+[tryivona]
+enabled=1
```

Comment:
This is still enabled, but if google fails, it would be better for the next option to be festival, as that is not optional.